docker/diffusers-pytorch-cpu/Dockerfile

```diff
@@ -36,6 +37,7 @@ RUN python3 -m pip install --no-cache-dir --upgrade pip && \
         numpy \
         scipy \
         tensorboard \
-        transformers
+        transformers \
+        librosa
```

Comment:
Ok for me @anton-l what do you think?

Reply:
Thanks for reviewing @patrickvonplaten !